### ISSUE TITLE
accessibility issue missing the example with label busy indicator

### ISSUE THREAD

**Issue by Lokanathan-k** [SAP/fundamental-styles#2338]
Description: There is no visual representation ‘please wait’ for the busy indicator as highlighted in red.

Expected: There is no visual representation "Please wait" for the busy indicator as highlighted in red 

Attached the screenshot
![image](https://user-images.githubusercontent.com/32538291/91549532-b57ae700-e944-11ea-8489-7e92ac8db56c.png)
 
expected output:
![image](https://user-images.githubusercontent.com/53534379/117546788-d81fdc80-b049-11eb-80b9-5f7e075543a2.png)

**Comment by Lokanathan-k** [comment]
closes as part of the PR:https://github.com/SAP/fundamental-styles/pull/2339
